lib/open_api_spex/test/test_assertions.ex

```diff
@@ -0,0 +1,42 @@
+defmodule OpenApiSpex.TestAssertions do
+  @moduledoc """
+  Defines helpers for testing API responses and examples against API spec schemas.
+  """
+  import ExUnit.Assertions
+  alias OpenApiSpex.OpenApi
+  alias OpenApiSpex.Cast.Error
+
+  @dialyzer {:no_match, assert_schema: 3}
+
+  @doc """
+  Asserts that `value` conforms to the schema with title `schema_title` in `api_spec`.
+  """
+  @spec assert_schema(map, String.t(), OpenApi.t()) :: map | no_return
+  def assert_schema(value = %{}, schema_title, api_spec = %OpenApi{}) do
+    schemas = api_spec.components.schemas
+    schema = schemas[schema_title]
+
+    if !schema do
```

Comment:
I would use `Map.fetch/2` for that instead of `Access` behaviour.

Reply:
`Map.fetch/2` doesn't provide what this needs. This looks for a truthy value. `Map.fetch/2` only returns an error if the key is missing.